### PR TITLE
crypto-org-wallet: init at version 0.1.1

### PR DESCRIPTION
**Pull request by Th0rgal**
###### Motivation for this change
Crypto.com recently released the crypto.org blockchain project. This application (still in beta) allows to create or access a CRO wallet.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by hugolgst** [comment]
@Th0rgal Please change your commit to `crypto-org-wallet: init at 0.0.24`.
See https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

**Comment by hugolgst** [comment]
Result of `nixpkgs-review pr 117497` [1](https://github.com/Mic92/nixpkgs-review)

**Review by hugolgst — CHANGES_REQUESTED** [review]
You did not register the package in https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix

**Comment by hugolgst** [comment]
Result of `nixpkgs-review pr 117497` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>crypto-org-wallet</li>
  </ul>
</details>

**Review by hugolgst — APPROVED** [review]

File: pkgs/applications/blockchains/crypto-org-wallet.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  appimageContents = appimageTools.extractType2 { inherit name src; };
+in appimageTools.wrapType2 rec {
+  inherit name src;
+
+  extraInstallCommands = ''
+    mv $out/bin/${name} $out/bin/${pname}
+    install -m 444 -D ${appimageContents}/${pname}.desktop $out/share/applications/${pname}.desktop
+    ${imagemagick}/bin/convert ${appimageContents}/${pname}.png -resize 512x512 ${pname}_512.png
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/crypto-org-wallet.nix:21]:
```suggestion
    convert ${appimageContents}/${pname}.png -resize 512x512 ${pname}_512.png
```

File: pkgs/applications/blockchains/crypto-org-wallet.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  src = fetchurl {
+    url =
+      "https://github.com/crypto-com/${pname}/releases/download/v${version}/${name}-x86_64.AppImage";
+    sha256 = "1v66hzikpww5mw563i16v9hp6320cxpyydigm6div7jbmw6n326h";
+  };
+
+  appimageContents = appimageTools.extractType2 { inherit name src; };
+in appimageTools.wrapType2 rec {
+  inherit name src;
+
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/crypto-org-wallet.nix:17]:
```suggestion
  inherit name src;

  nativeBuildInputs = [ imagemagick ];

```

**Comment by Th0rgal** [comment]
@SuperSandro2000 your changes seem to break the package:
```
error: anonymous function at /nix/store/ih14z41892v89vcza40js0djmdly41zc-nixos-20.09.3613.34f85de51bb/nixos/pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix:3:1 called with unexpected argument 'nativeBuildInputs', at /nix/store/ih14z41892v89vcza40js0djmdly41zc-nixos-20.09.3613.34f85de51bb/nixos/pkgs/build-support/build-fhs-userenv-bubblewrap/default.nix:16:9
(use '--show-trace' to show detailed location information)

There are 46 unread and relevant news items.
Read them by running the command 'home-manager news'.
```

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 your changes seem to break the package:
> 
> ```
> error: anonymous function at /nix/store/ih14z41892v89vcza40js0djmdly41zc-nixos-20.09.3613.34f85de51bb/nixos/pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix:3:1 called with unexpected argument 'nativeBuildInputs', at /nix/store/ih14z41892v89vcza40js0djmdly41zc-nixos-20.09.3613.34f85de51bb/nixos/pkgs/build-support/build-fhs-userenv-bubblewrap/default.nix:16:9
> (use '--show-trace' to show detailed location information)
> 
> There are 46 unread and relevant news items.
> Read them by running the command 'home-manager news'.
> ```

well, appimageTools.wrapType2 does not support cross compiling I guess.

File: pkgs/applications/blockchains/crypto-org-wallet.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{ lib, fetchurl, makeDesktopItem, appimageTools, imagemagick }:
+
+let
+  pname = "chain-desktop-wallet";
+  version = "0.1.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url =
+      "https://github.com/crypto-com/${pname}/releases/download/v${version}/${name}-x86_64.AppImage";
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/crypto-org-wallet.nix:10]:
```suggestion
    url = "https://github.com/crypto-com/${pname}/releases/download/v${version}/${name}-x86_64.AppImage";
```

**Pull request merged**